index.html

```diff
@@ -0,0 +1,209 @@
+        </p>
+        <time datetime="2023-11-20T19:30">26 Nov 2023 @ 3:00pm</time>
+        <p class="location">
+          TREM International Hqtrs. Gbagada Express way, Lagos.
+        </p>
+      </section>
+    </main>
+    <!-- Main-ends -->
+
+    <!-- program-starts -->
+    <section class="program">
+      <div class="container">
+        <h2>Main Program</h2>
+        <hr />
+
+        <div class="program-contents">
+          <div class="content">
+            <div class="icon-title">
+              <i class="uil uil-analysis"></i>
+              <h3>Trends</h3>
+            </div>
+            <p>Level up your brand with the latest digital marketing trends.</p>
+          </div>
+
+          <div class="content">
+            <div class="icon-title">
+              <i class="uil uil-tv-retro"></i>
+              <h3>Brands</h3>
+            </div>
+            <p>Branding strategies in the age of the Metaverse</p>
+          </div>
+
+          <div class="content">
+            <div class="icon-title">
+              <i class="uil uil-comment-alt-notes"></i>
+              <h3>Forum</h3>
+            </div>
+            <p>
+              Have the time to share your thoughts and opinions with experts for
+              each topic.
+            </p>
+          </div>
+
+          <div class="content">
+            <div class="icon-title">
+              <i class="uil uil-presentation-edit"></i>
+              <h3>Works</h3>
+            </div>
+            <p>
+              Try creating your own work using open source license rather than
+              just watching.
+            </p>
+          </div>
+
+          <div class="content">
+            <div class="icon-title">
+              <i class="uil uil-palette"></i>
+              <h3>Design</h3>
+            </div>
+            <p>
+              Get the opportunity to connect with different brands to showcase
+              your creative prowess.
+            </p>
+          </div>
+
+          <div class="join">
+            <a href="#">Join KLWC 2023.</a>
+          </div>
+        </div>
+        <p class="show">SEE THE WHOLE PROGRAM</p>
+      </div>
+    </section>
+    <!-- programs-ends -->
+
+    <!-- speakers-starts -->
+    <section class="speakers">
+      <div class="container">
+        <h2>Featured Speakers</h2>
+        <hr />
+
+        <div class="featured-speakers"></div>
+      </div>
+    </section>
+    <!--Speakers-ends -->
+
+    <!-- partners-starts -->
+    <section class="partner sponsor" id="sponsor">
+      <div class="container">
+        <h4>Partners</h4>
+        <hr />
+        <div class="logos logo-home">
+          <p class="sponsor1">Dangote</p>
+          <img src="./images/sponsor-2.png" alt="" />
+          <img src="./images/sponsor-3.png" alt="" />
+          <img src="./images/sponsor-4.png" alt="" />
+          <img src="./images/sponsor-5.png" alt="" />
+          <img src="./images/gtv.png" alt="" />
```

Comment:
- [ ] Good job so far 👏 As suggested by the previous reviewer, please add the alt description text within the `alt` tags for easy accessibility 😃